cmd/argocd/commands/app.go

```diff
@@ -1538,6 +1547,7 @@ func NewApplicationSyncCommand(clientOpts *argocdclient.ClientOptions) *cobra.Co
 	command.Flags().StringArrayVar(&infos, "info", []string{}, "A list of key-value pairs during sync process. These infos will be persisted in app.")
 	command.Flags().BoolVar(&diffChangesConfirm, "assumeYes", false, "Assume yes as answer for all user queries or prompts")
 	command.Flags().BoolVar(&diffChanges, "preview-changes", false, "Preview difference against the target and live state before syncing app and wait for user confirmation")
+	command.Flags().StringArrayVar(&projects, "project", []string{}, "Sync apps that belongs to the specified projects.")
```

Comment:
Is this comma-delimited, or is the user expected to pass `--project` multiple times? I'd specify that in the doc string.

Reply:
Need to pass multiple times. I've updated the help string